src/main/java/uk/gov/ida/verifyserviceprovider/services/NonMatchingAssertionService.java

```diff
@@ -123,4 +124,18 @@ private TranslatedResponseBody translateAssertions( List<Assertion> assertions )
     public TranslatedResponseBody translateNonSuccessResponse( StatusCode statusCode ) {
         return null;
     }
```

Comment:
What's the plan for non-success responses?  We should discuss this with Andy.

Reply:
Yes, I think we should or maybe we are focusing on the happy path for now

Reply:
@andy-paine will raise a card to deal with this at some point